runtime/v2/runner/src/main/java/com/walmartlabs/concord/runtime/v2/runner/script/DefaultScriptEvaluator.java

```diff
@@ -37,153 +37,184 @@
 import java.io.BufferedWriter;
```

Comment:
TOP OF REVIEW - stop! review using GitHub whitespace ignore mode:

https://github.com/walmartlabs/concord/pull/709/files?w=1